code/workspaces/infrastructure-api/src/config/routes.js

```diff
@@ -16,6 +17,7 @@ function configureRoutes(app) {
   app.use('/volume', volumeRouter);
   app.use('/volumes', volumesRouter);
   app.use('/logs', logsRouter);
+  app.use('/userresources', userResourcesRouter);
```

Comment:
Not sure if `user-resources` would be fractionally clearer? (very minor!)

Reply:
Agreed, changed.